view/frontend/web/instantsearch.js

```diff
@@ -207,408 +206,454 @@ requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBu
 							var content = '<div class="no-results">';
 							content += '<div><b>' + algoliaConfig.translations.noProducts + ' "' + $("<div>").text(data.results.query).html() + '</b>"</div>';
 							content += '<div class="popular-searches">';
-							
+
 							if (algoliaConfig.showSuggestionsOnNoResultsPage && this.suggestions.length > 0) {
 								content += '<div>' + algoliaConfig.translations.popularQueries + '</div>' + this.suggestions.join(', ');
 							}
-							
+
 							content += '</div>';
 							content += algoliaConfig.translations.or + ' <a href="' + algoliaConfig.baseUrl + '/catalogsearch/result/?q=__empty__">' + algoliaConfig.translations.seeAll + '</a>'
-							
+
 							content += '</div>';
-							
-							$('#instant-search-results-container').html(content);
+
+							$('#instant-empty-results-container').html(content);
 						}
 					}
 				}
 			],
 			/**
-			 * Search box
-			 * Docs: https://community.algolia.com/instantsearch.js/v2/widgets/searchBox.html
+			 * searchBox
+			 * Docs: https://www.algolia.com/doc/api-reference/widgets/search-box/js/
 			 **/
 			searchBox: {
 				container: instant_selector,
-				placeholder: algoliaConfig.translations.searchFor
+				placeholder: algoliaConfig.translations.searchFor,
+				showSubmit: false
 			},
 			/**
-			 * Stats
-			 * Docs: https://community.algolia.com/instantsearch.js/v2/widgets/stats.html
+			 * stats
+			 * Docs: https://www.algolia.com/doc/api-reference/widgets/stats/js/
 			 **/
 			stats: {
 				container: '#algolia-stats',
 				templates: {
-					body: $('#instant-stats-template').html()
-				},
-				transformData: function (data) {
-					data.first = data.page * data.hitsPerPage + 1;
-					data.last = Math.min(data.page * data.hitsPerPage + data.hitsPerPage, data.nbHits);
-					data.seconds = data.processingTimeMS / 1000;
-					
-					data.translations = window.algoliaConfig.translations;
-					
-					return data;
+					text: function (data) {
+						var hoganTemplate = algoliaBundle.Hogan.compile($('#instant-stats-template').html());
+						var _data = Object.assign({}, data)
```

Comment:
`Object.assign` is not supported on IE. Is it transpiled in the environment?

Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

Reply:
No it's not!, thanks for pointing in out!